ingestion/src/metadata/utils/sql_lineage.py

```diff
@@ -116,19 +158,50 @@ def _create_lineage_by_table_name(
         logger.error(err)
 
 
+def poplate_column_lineage_map(raw_column_lineage):
```

Comment:
typo!

Reply:
we should add some unit tests covering this function, a lot of business thingies going on here